pages/test_analytics/public_test_suites.md

```diff
@@ -0,0 +1,24 @@
+# Public test suites
+
+If you're working on an open-source project, and want the whole world to be able to see your test analytics, you can make your test suite public.
+
+
+Making a suite public provides read-only public/anonymous access to:
+
+- All test suite data
+- Run results
+- Tests
+- Test executions
+- Test execution span data
+- Test suite environmental variables
```

Comment:
I'm pretty sure "span data" is a buildkite concept, and it doesn't apply to all users so could be good to be more specific here. Maybe something closer to:
```suggestion
- Test execution data. For those using buildkites ruby test collector this includes SQL query data, HTTP request paths and the execution timeline for test-collector-ruby users 
- Run environmental variables
```

Reply:
I pulled this from our confirmation popup to make suites public, would we also need to update that there?
<img width="368" alt="Screenshot 2023-11-29 at 11 25 34 AM" src="https://github.com/buildkite/docs/assets/25017977/62447ba5-2ed9-4317-8ec5-60580f17b69d">

Reply:
Hm, not sure? Keen for @jasmine-q's take, as the variables relate to a run, not a suite per se.